package.json

```diff
@@ -84,5 +86,8 @@
   },
   "optionalDependencies": {
     "node-sass": "^4.12.0"
-  }
+  },
+  "browserslist": [
+    "last 2 versions"
```

Comment:
I think we've said that we don't support IE in the past, so I'd add `not ie`

Reply:
some data, IE is 0.42% of our traffic over the last week and I honestly feel like a certain portion of that has to be developers just checking to see if the site works in IE :P

Reply:
Done